channel/src/main/java/com/synopsys/integration/alert/channel/email/actions/EmailDistributionTestAction.java

```diff
@@ -62,10 +62,12 @@ public MessageResult testConfig(
     private DistributionJobModel creatUpdatedJobModelWithEmailAddresses(DistributionJobModel originalJobModel, @Nullable String destination) throws IntegrationException {
         Set<String> updateEmailAddresses = emailActionHelper.createUpdatedEmailAddresses(originalJobModel, destination);
         EmailJobDetailsModel originalEmailJobDetails = originalJobModel.getDistributionJobDetails().getAsEmailJobDetails();
+
+        // For testing configuration, just use additional email addresses field
```

Comment:
This does not seem correct, why are we only using the additional email addresses? Part of the email distribution test is to verify the email recipients. So we should be using the configuration as specified in the test.

Reply:
`EmailTestActionHelper` will get all of the correct recipients for a Job. 

When testing configuration, the event is not tied to a real notification, so when the event gets to the channel, the `EmailAddressHandler` will not be able to look up the "Project Owner" or "project members" correctly. Before my changes, we would shove an extra field into `FieldUtility` with the results of `EmailTestActionHelper`; now that there is no `FieldUtility` we append the results of `EmailTestActionHelper` as additional email addresses instead and only use those emails for testing.